Dockerfile

```diff
@@ -1,11 +1,10 @@
 FROM ubuntu:trusty
 ADD . /code
 WORKDIR /code
-RUN apt-get update
-RUN apt-get install -y git software-properties-common python python-dev python-pip
-RUN add-apt-repository -y ppa:chris-lea/node.js
-RUN apt-get update
-RUN apt-get install -y nodejs
-RUN npm install bower
-RUN pip install -r requirements.txt
+ENV DEBIAN_FRONTEND noninteractive
+RUN add-apt-repository -y ppa:chris-lea/node.js && \
```

Comment:
... also, you had some `-y` missing and `pip` was not installing.  I had fixed all those and I'll merge.